Rocket.Chat/Views/ConfigTable/ConfigTableCellTextFieldCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ConfigTableCellTextFieldCell.swift
+//  Rocket.Chat
+//
+//  Created by Bruno Macabeus Aquino on 28/09/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+class ConfigTableCellTextFieldCell: UITableViewCell, ConfigTableCellProtocol {
+    static let identifier = "kConfigTableCellTextField"
+    static let xibFileName = "ConfigTableCellTextFieldCell"
+    static let defaultHeight: Float = 40
+    weak var delegate: ConfigTableCellDelegate?
+    var key: String?
+
+    @IBOutlet weak var imgRoomIcon: UIImageView!
+    @IBOutlet weak var textFieldInput: UITextField!
+    @IBOutlet weak var imgClearText: UIImageView!
```

Comment:
I honestly think that this cell `ConfigTableCellTextFieldCell` having properties like `imgRoomIcon` and `imgClearText` will cause us many problems in the future. I prefer to go the simple way and just create a cell class to each specifically cell we need right now, except for the ones you see cases that you can **really** reuse. This cell is being used only once and is looks very over engineered.